cpp/include/cudf_test/base_fixture.hpp

```diff
@@ -244,7 +244,11 @@ inline auto make_managed() { return std::make_shared<rmm::mr::managed_memory_res
 
 inline auto make_pool()
 {
-  return rmm::mr::make_owning_wrapper<rmm::mr::pool_memory_resource>(make_cuda());
+  auto const [free, total] = rmm::detail::available_device_memory();
+  auto min_alloc =
+    rmm::detail::align_down(std::min(free, total / 10), rmm::detail::CUDA_ALLOCATION_ALIGNMENT);
```

Comment:
Are we specifying this now because the default pool size rmm picks of half the available memory restricts the total parallelism available to the cudf test suite?

Reply:
Exactly the reason.

Reply:
Due to the round-up on the 50% it generally causes failures even when running two tests at the same time.

Reply:
Cool makes sense.